R/bundle.R

```diff
@@ -80,7 +82,24 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
   bundlePath
 }
 
-isShinyRmd <- function(filename) {
+appHasParameters <- function(appDir, files) {
+  rmdFiles <- grep("^[^/\\\\]+\\.rmd$", files, ignore.case = TRUE, perl = TRUE,
```

Comment:
Gotcha. Maybe we just want to scan the primary Rmd for now, then, but not a big deal either way.